src/server/services/wms/qgswmsparameters.cpp

```diff
@@ -85,6 +86,37 @@ namespace QgsWms
     return val;
   }
 
+  QString QgsWmsParameter::loadUrl() const
+  {
+    // Check URL -- it will be used in error messages
+    const QUrl url = toUrl();
+
+    bool ok = false;
+    const QString content = QgsServerParameterDefinition::loadUrl( ok );
+
+    if ( !ok )
+    {
+      const QString msg = QString( "%1 request error for %2" ).arg( name( mName ), url.toString() );
+      QgsMessageLog::logMessage( msg );
```

Comment:
tr(...)

Reply:
The message is also raised and be displayed in an OGC Exceptions. You wrote in #8006 discussion that messages in OGC Exceptions does not have to be localized.

I can removed the logMessage because QgsServerParameterDefinition already logs messages.
What do you think ?

Reply:
> The message is also raised and be displayed in an OGC Exceptions. You wrote in #8006 discussion that messages in OGC Exceptions does not have to be localized.

Indeed.

> I can removed the logMessage because QgsServerParameterDefinition already logs messages.
What do you think ?

Indeed :)

Reply:
done @pblottiere thanks!